COLLABORATOR_GUIDE.md

```diff
@@ -75,47 +78,86 @@ Collaborators or additional evidence that the issue has relevance, the
 issue may be closed. Remember that issues can always be re-opened if
 necessary.
 
+### Ready to land pull requests
+
+A pull request that is still awaiting the minimum review time is considered
+`ready` as soon as the CI has been started, it has at least one approval, and it
+has no outstanding review comments. Please always make sure to add the
+appropriate `ready` label to the PR in that case and remove it again as soon as
+that condition is not met anymore.
```

Comment:
I thought `ready` was ok to land, so CI has run and someone has checked if this could land. So `ready` should mean "ready to land", not "check CI because this could be ready to land".

Also, it should include a CITGM run for semver-major.

Reply:
@mcollina When I introduced that label, I didn’t intend it to have that meaning.

Practically speaking, it’s easier this way; you can kick off CI and add the label, because you’re planning to go through the list of open PRs with that label, and can then check CI.

In particular, it avoids having to keep track of which PRs have CIs started and which not.

Reply:
As @addaleax explained it is meant to be the way it is described right now. This has helped me (and probably others just as much, but I can not speak for others) quite a lot already to keep a better overview of PRs that should be checked again.
If everything was already checked there is little point besides the waiting time to have the label at all because in that case the PR should just be merged instead.

Reply:
I agree that it is easier and it's what we want, and it's probably the right moment to apply a tag. However, I don't think a PR is "ready to land" if it does not have CI passing, so the "ready" term is very misleading to me: I've landed something that should not have landed because of it. It's definitely not ready to land if CI has not been checked for which failures are infrastructure or not.

Reply:
I'd have to agree with @mcollina on this. Perhaps `ready` in this case should really mean `ready to consider landing`. Before the PR can actually land, a look at the CI results is required.